Dockerfile

```diff
@@ -19,51 +19,56 @@ LABEL org.opencontainers.image.documentation="https://hyp3-docs.asf.alaska.edu"
 ARG DEBIAN_FRONTEND=noninteractive
 ENV PYTHONDONTWRITEBYTECODE=true
 
-RUN apt-get update && apt-get upgrade -y && \
-    apt-get install -y --no-install-recommends \
-    build-essential curl gcc gdal-bin \
-    gimp gnuplot gnuplot-data gnuplot-qt libblas-dev libblas3 \
-    libfftw3-dev libgdal-dev libgdal20 \
-    libgtk2.0-bin libgtk2.0-common libgtk2.0-dev \
-    libhdf5-100 libhdf5-dev liblapack-dev liblapack3 \
-    python3-dev python3-h5py python3-netcdf4 \
-    python3-matplotlib python3-pip python3-scipy tcsh unzip vim wget git && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* \
-    && pip3 install --no-cache-dir --upgrade pip setuptools wheel
-
-COPY GAMMA_SOFTWARE-20191203 /usr/local/GAMMA_SOFTWARE-20191203/
-
-RUN export CPLUS_INCLUDE_PATH=/usr/include/gdal && \
-    export C_INCLUDE_PATH=/usr/include/gdal && \
-    python3 -m pip install --no-cache-dir GDAL==2.2.3
+RUN apt update \
+    && apt upgrade -y \
+    && apt install -y --no-install-recommends \
+         # GAMMA requirements per sections 2-6, 9 of the Linux installation guide
+         libfftw3-3 libfftw3-dev libfftw3-single3 \
+         gnuplot gnuplot-data gimp \
+         gdal-bin libgdal-dev \
+         libhdf5-dev libhdf5-103 \
+         libblas-dev libblas3 liblapack-dev liblapack3 liblapack-doc \
+         python-is-python3 python3-numpy python3-matplotlib python3-scipy python3-shapely \
+         # GAMMA scripts require csh/tcsh
+         tcsh \
+         # Additional installs
+         python3-pip wget git vim \
+    && apt clean \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY GAMMA_SOFTWARE-20210701 /usr/local/GAMMA_SOFTWARE-20210701/
 
 COPY . /hyp3-gamma/
-RUN  python3 -m pip install --no-cache-dir /hyp3-gamma \
+RUN python3 -m pip install --no-cache-dir /hyp3-gamma \
     && rm -rf /hyp3-gamma
 
 ARG CONDA_GID=1000
 ARG CONDA_UID=1000
 
-RUN groupadd -g "${CONDA_GID}" --system conda && \
-    useradd -l -u "${CONDA_UID}" -g "${CONDA_GID}" --system -d /home/conda -m  -s /bin/bash conda
+RUN groupadd -g "${CONDA_GID}" --system conda \
+    && useradd -l -u "${CONDA_UID}" -g "${CONDA_GID}" --system -d /home/conda -m -s /bin/bash conda
 
 USER ${CONDA_UID}
 SHELL ["/bin/bash", "-l", "-c"]
 ENV PYTHONDONTWRITEBYTECODE=true
-ENV GAMMA_HOME=/usr/local/GAMMA_SOFTWARE-20191203
+
+# GAMMA environment variables per section 1 of the Linux installation guide
+ENV GAMMA_HOME=/usr/local/GAMMA_SOFTWARE-20210701
 ENV MSP_HOME=$GAMMA_HOME/MSP
 ENV ISP_HOME=$GAMMA_HOME/ISP
 ENV DIFF_HOME=$GAMMA_HOME/DIFF
 ENV DISP_HOME=$GAMMA_HOME/DISP
 ENV LAT_HOME=$GAMMA_HOME/LAT
 ENV PATH=$PATH:$MSP_HOME/bin:$ISP_HOME/bin:$DIFF_HOME/bin:$LAT_HOME/bin:$DISP_HOME/bin
-ENV PATH=$PATH:$MSP_HOME/scripts:$ISP_HOME/scripts:$DIFF_HOME/scripts:$LAT_HOME/scripts
-ENV GAMMA_RASTER=BMP
+ENV PATH=$PATH:$MSP_HOME/scripts:$ISP_HOME/scripts:$DIFF_HOME/scripts:$LAT_HOME/scripts:$DISP_HOME/scripts
+ENV OS=linux64
+ENV PYTHONPATH=.:$GAMMA_HOME:$PYTHONPATH
```

Comment:
Why is the `.` (current working directory) needed? Usually this is something you want to avoid.